backend/pluto/pluto/_internal/server/flask_server.py

```diff
@@ -52,17 +57,56 @@ def configs(self, config: Config):
             self.app.config[config_name.upper()] = value
 
     def _add_endpoints(self):
+        self._add_expense_endpoints()
+        self._add_income_endpoints()
+        self._add_user_endpoints()
+
+    def _add_expense_endpoints(self):
+        self.add_endpoint(
```

Comment:
Acho que você criou dois endpoints para a mesma rota `/expenses/`. Vi que um é GET e outro é para POST. Isso funciona?

Reply:
It works :) . That's the usual behavior of HTTP APIs. It's just a single HTTP resource with two allowed methods.